src/Helpers/Helpers.php

```diff
@@ -8,4 +8,19 @@ public static function checkIfNullValue($value): bool
     {
         return !is_null($value);
     }
+
+    public static function buildUrlWithQueryParams(string $url, array $params = []): string
+    {
+        if (empty($params)) {
+            return $url;
+        }
+
+        if (parse_url($url, PHP_URL_QUERY)) {
+            $separator = '&';
+        } else {
+            $separator = '?';
+        }
+
+        return $url.$separator.http_build_query($params, '');
```

Comment:
do you know why there was a default separator in the code? `http_build_query($auth_params, '', '&');`?

Reply:
When this change was introduced, I didn't read it carefully enough to notice that the separator we were providing was the default value specified by the function. We don't need it to do anything custom here, so there's no reason for us to override the default. Basically, it never should have had it, but I missed that on my initial review.